src/app/problems/_test/mock.ts

```diff
@@ -5,6 +5,53 @@ import {McqFormData} from "@app/problems/_forms/mcq.form"
 import {JavaFormData} from "@app/problems/_forms/java.form"
 import {ParsonsFormData} from "@app/problems/_forms/parsons.form"
 import {UserStats} from "@app/_models/user_difficulty_stats"
+import {Goal, GoalItem} from "@app/_models/goal/goal"
+
+export const MOCK_GOAL_ITEM: GoalItem = {
```

Comment:
This is in the wrong mock file.
This file is in `.../problems/_test/mock.ts`
it should be in `.../problems/_test/mock.ts`